cmd/fleet/serve.go

```diff
@@ -722,6 +722,21 @@ func cronVulnerabilities(
 			sentry.CaptureException(err)
 		}
 
+		if err := ds.CleanUpUnusedSoftware(ctx); err != nil {
+			level.Error(logger).Log("msg", "cleaning up unused software", "err", err)
+			sentry.CaptureException(err)
+		}
+
+		// It's important vulnerabilities.PostProcess runs after ds.CleanUpUnusedSoftware
+		// because it cleans up any software that's not installed on the fleet (e.g. hosts removal,
+		// or software being uninstalled on hosts).
+		if !vulnDisabled {
+			if err := vulnerabilities.PostProcess(ctx, ds, vulnPath, logger, config); err != nil {
```

Comment:
What do you folks think of adding a `disable_post_processing` or `disable_centos_post_processing` config.

Reply:
Why would we allow for disabling that?

Reply:
Troubleshooting mostly, a bug in the feature in some deployments, etc.

Reply:
Hm, I think this is a key part of detection, I would just make sure we log enough in debug mode to troubleshoot.